comp/dogstatsd/server/server_test.go

```diff
@@ -503,7 +503,8 @@ func TestUDPForward(t *testing.T) {
 	requireStart(t, deps.Server)
 
 	conn, err := net.Dial("udp", deps.Server.UDPLocalAddr())
-	require.NoError(t, err, "cannot connect to DSD socket")
+	assert.NoError(t, err)
+	require.NotNilf(t, conn, "Failed to connect to UDP local address %s", err)
```

Comment:
Wh format the message manually here? `require.NoError` can report the message just fine by itself.